charts/puris/values.yaml

```diff
@@ -431,6 +431,7 @@ backend:
     dtr:
       # --Endpoint for DTR
       url: http://localhost:4243
+    generatematerialcatenaxid: true
```

Comment:
Please add documentation above the property in the format `# -- <documentation what this flag does` so that helm-test will use this description for the Readme

Description can be the same as your comment in the application.properties. Maybe it makes sense to put that as a note to the [admin guide](https://github.com/eclipse-tractusx/puris/blob/main/docs/adminGuide/Admin_Guide.md#onboard-master-data) chapter like "As the puris material represents a PartType twin, a catena-x id is mandatory. When providing material data, one can let the application generate the catena-x id automatically using the property `generatematerialcatenaxid` in the respective deployment."

Reply:
Done